benchmarks/oss_fuzz_benchmark_integration.py

```diff
@@ -96,6 +99,34 @@ def copy_oss_fuzz_files(project, commit_date, benchmark_dir):
         oss_fuzz_repo_manager.git(['reset', '--hard'])
 
 
+def add_oss_fuzz_corpus(project, fuzz_target, benchmark_dir):
+    """Add latest public corpus from OSS-Fuzz as the seed corpus."""
+    if not fuzz_target.startswith(project):
+        full_fuzz_target = '%s_%s' % (project, fuzz_target)
+    else:
+        full_fuzz_target = fuzz_target
+
+    corpus_backup_url = CORPUS_BACKUP_URL_FORMAT.format(
+        project=project, fuzz_target=full_fuzz_target)
+    command = ['gsutil', 'ls', corpus_backup_url]
+    result = subprocess.run(command, stdout=subprocess.PIPE)
+    if result.returncode:
+        raise Exception(f'No corpus found for project {project}.')
+
+    archive_filename = f'{fuzz_target}_seed_corpus.zip'
```

Comment:
Do you think we should call this seed corpus or something else?

Reply:
Done, made it oss_fuzz_corpus.zip